bin/gollum

```diff
@@ -197,8 +197,8 @@ end
 # Read command line options into `options` hash
 begin
   opts.parse!
-rescue OptionParser::InvalidOption
-  puts 'gollum: #{$!.message}'
+rescue OptionParser::InvalidOption => e
+  puts "gollum: #{$!.message}"
```

Comment:
If you're capturing the error as `e`, you might as well use `e` instead of `$!`.